test/test.js

```diff
@@ -292,16 +293,67 @@ describe('onclickoutside hoc', function() {
   });
 
   describe('with child rendering as null', function() {
-    var StatelessComponent = () => null;
-
-    it('should throw an error when wrapped component renders as null', function() {
+    it('shouldn\'t throw an error when wrapped SFC renders as null', function() {
+      var StatelessComponent = () => null;
       try {
         wrapComponent(StatelessComponent);
-        assert(false, 'component was wrapped despite having no DOM node on mount');
-      } catch(e) {
-        assert(true, 'an error was thrown');
+        assert(true, 'component was wrapped despite having no DOM node on mount');
+      } catch (err) {
+        assert(false, 'an error was thrown');
       }
     });
-  });
 
+    var counter;
```

Comment:
this code should really be in a `before` block rather than persisting from one test to the next; tests should be able to run regardless of the order they run in. We can land it this way, but we'll need a follow-up to clean up this code so that tests are self-contained.